config/default_records/plate_types/plate_types.yml

```diff
@@ -5,6 +5,8 @@ ABgene_0800:
   maximum_volume: 180
 FluidX075:
   maximum_volume: 500
+FluidX05:
```

Comment:
So in the story requirements it specifically asks for the name 'FluidX 0.5mL'. Are they ok with this 'FluidX05' name or am I not seeing where the display name is set?